commit 63da6900b0c623bf0b591cd42ae601cdde96aaec
Merge: a18b625 2701dd8
Author: Bo Herrmannsen <bo.herrmannsen@gmail.com>
Date:   Fri Jan 2 15:35:40 2015 +0100

    Merge pull request #1282 from vandarin/Development
    
    Due to the fix for #1248, X and Y probe offsets must not be floats.

diff --git a/Marlin/Configuration.h b/Marlin/Configuration.h
index 980efc5..fe6e371 100644
--- a/Marlin/Configuration.h
+++ b/Marlin/Configuration.h
@@ -392,6 +392,7 @@ const bool Z_MAX_ENDSTOP_INVERTING = true; // set to true to invert the logic of
 
 
   // these are the offsets to the probe relative to the extruder tip (Hotend - Probe)
+  // X and Y offsets must be integers
   #define X_PROBE_OFFSET_FROM_EXTRUDER -25
   #define Y_PROBE_OFFSET_FROM_EXTRUDER -29
   #define Z_PROBE_OFFSET_FROM_EXTRUDER -12.35
